### ISSUE TITLE
Window behind/overlapped with auto-hidden taskbar

### ISSUE THREAD

**Issue by yur1xpp** [thura10/airwave#18]
The window overlapped with the taskbar if it was set to auto-hide (Taskbar settings->Automatically hide the taskbar in desktop mode). Is it possible to fix this? Thanks in advance!

![image](https://user-images.githubusercontent.com/34767893/120928219-4ffb2880-c716-11eb-9f8f-7196837f41c2.png)

**Comment by thura10** [comment]
If you click on the taskbar icon to hide the window, then click again to reshow it, the window should be displayed above the taskbar. Let me know if that works.

**Comment by yur1xpp** [comment]
Yeah that works, but would love if it just appear above the taskbar (like how if the taskbar was not set to hidden), since to activates it, the taskbar needs to be appeared beforehand, and disappeared right after mouse was hover away (when interacting with airwave windows), and makes the windows being overlapped.
